components/SummaryStats/SummaryStats.tsx

```diff
@@ -20,6 +22,8 @@ interface SummaryStatsViewModel {
   stats: IRow[]
 }
 
+const [useTickerSelection] = bind(tickerSelections$, "")
```

Comment:
You already have a hook for this.  Just add to exports from TickerSearch

Reply:
👌

Reply:
@dwalker11 - my bad.  You are right.  It doesn't matter at the moment because this is the only part of the app that's using this hook, but, since you're not doing any transformations on the underlying stream, it might make sense to define the hook and export it from TickerSearch, so that we don't inadvertently do the bind in each consumer component and duplicate.  But it's a judgment call.